core/src/main/java/com/netflix/conductor/core/events/SimpleActionProcessor.java

```diff
@@ -196,19 +196,29 @@ private Map<String, Object> startWorkflow(
         Map<String, Object> output = new HashMap<>();
         try {
             Map<String, Object> inputParams = params.getInput();
+            inputParams.put("workflowName", params.getName());
+            inputParams.put("version", "${version}");
+            inputParams.put("params", "${params}");
+
```

Comment:
Allow an event message to start a specific workflow by passing workflow name / version